New package: SpinSim v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: SpinSim
- Repository: https://github.com/javinoram/SpinSim.jl
- Created by: @javinoram
- Version: v0.1.0
- Commit: bb556839c085c4e3d22fff634a45d16dce5f3f02
- Reviewed by: @javinoram
- Reference: https://github.com/javinoram/SpinSim.jl/issues/16#issuecomment-2496130640
- Description: Julia package to simulate and study spin models
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
Hamiltonian module implemented
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a8359802cf15fc4380874531f483c58cfa4d0adcba112a7da2353c2dc807bd959c6bf4e9858e7acfd1b9752715ac7ff8370789c8deec57cdb1b9c339580a37064c963c7d79344846f1c2401d0bfd1bf57dc5ce267c88afc3702791d07cf778d568a2ddf40a2150c2021e170faeef50bf3885df56a1965684de015d38474ee0f0c10022 -->

goerz commented:
`[noblock]` Documenter` and `TestItemRunner` should not be dependencies of your package. Make sure to only have the minimum dependencies required for the code itself. The environment for tests and documentation building are separate and must manage their dependencies separately.

javinoram commented:
Now it should be ok.

goerz commented:
[noblock] You still need to add the compat specification for Julia (probably simply `1.10`), and you should also set the compat for `LinearAlgebra` to  `1`. And don’t forget to add `[noblock]` to all your comments.

JuliaRegistrator committed:
New package: SpinSim v0.1.0

UUID: a9872f58-4f48-11ef-0c06-ff7f93bd3947
Repo: https://github.com/javinoram/SpinSim.jl.git
Tree: fd249f8ea57cf9ebf3bef86b71731c5dfda301a2

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

javinoram commented:
[noblock] I added the "compat" section in the main .toml, now I'm going to see the TagBot for future usage

javinoram commented:
[noblock] Hi, I added the tagbot so, I will keep an eye out if I'm missing something else.
